tools/stylelint-rules/comment-no-loud.js

```diff
@@ -0,0 +1,42 @@
+// It's fun to write JavaScript
+const stylelint = require('stylelint');
```

Comment:
We should at least add a refer to the original code, so we can come back to it in the future. 
It's also fair to drop this custom code for now (we don't need it today), and suggest this feature to the lib owner.